src/components/NavigationBar/NavigationBar.tsx

```diff
@@ -105,12 +110,14 @@ export default function NavigationBar() {
               >
                 Development Team
               </MenuItem>
-              <MenuItem
-                className="flex w-full text-start"
-                icon={<FontAwesomeIcon icon={faToolbox} />}
-              >
-                Administration
-              </MenuItem>
+              <Link to={`administration`}>
+                <MenuItem
+                  className="flex w-full text-start"
+                  icon={<FontAwesomeIcon icon={faToolbox} />}
+                >
+                  Administration
+                </MenuItem>
+              </Link>
```

Comment:
Could we do this in a for loop to remove duplicate code?
Like have an array objects where each object would contain the date for each menu item and even handle submenus as well.